selftests/unit/test_replay.py

```diff
@@ -3,7 +3,7 @@
 import unittest
 
 from avocado.core import test
-from avocado.plugins import replay
+from avocado.plugins import replay_legacy
```

Comment:
what about creating an `avocado.plugins.legacy` package, to avoid getting a polluted directory?